apps/brainstorm/build.sh

```diff
@@ -10,7 +10,8 @@ OUTPUT="type=${OUTPUT:-docker}"
 
 # Tip: use `BUILDKIT_PROGRESS=plain` to see more.
 
-exec docker buildx build \
+cp -r ../../core ./core
+docker buildx build \
```

Comment:
by removing the `exec` we'll be losing the return code from this call. Just add a `set -e` near the beginning.

Reply:
🫡